sdk/core/Azure.Core.Amqp/api/Azure.Core.Amqp.netstandard2.0.cs

```diff
@@ -19,14 +19,16 @@ namespace Azure.Core.Amqp
     public partial class AmqpAnnotatedMessage
     {
         public AmqpAnnotatedMessage(Azure.Core.Amqp.AmqpMessageBody body) { }
-        public System.Collections.Generic.IDictionary<string, object> ApplicationProperties { get { throw null; } }
+        public System.Collections.Generic.IDictionary<string, object?> ApplicationProperties { get { throw null; } }
         public Azure.Core.Amqp.AmqpMessageBody Body { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, object> DeliveryAnnotations { get { throw null; } }
-        public System.Collections.Generic.IDictionary<string, object> Footer { get { throw null; } }
+        public System.Collections.Generic.IDictionary<string, object?> DeliveryAnnotations { get { throw null; } }
```

Comment:
The nullability of all the dictionaries was not correct. The object is allowed to be null, per SB service behavior and AMQP spec.